bin/fetch-recent-miner-measurements.js

```diff
@@ -0,0 +1,214 @@
+// dotenv must be imported before importing anything else
+import 'dotenv/config'
+
+import { Point } from '@influxdata/influxdb-client'
+import * as Sentry from '@sentry/node'
+import createDebug from 'debug'
+import fs from 'node:fs'
+import { mkdir, readFile, writeFile } from 'node:fs/promises'
+import os from 'node:os'
+import path from 'node:path'
+import pMap from 'p-map'
+import { IE_CONTRACT_ADDRESS } from '../lib/config.js'
+import { createMeridianContract } from '../lib/ie-contract.js'
+import { fetchMeasurements, preprocess } from '../lib/preprocess.js'
+import { RoundData } from '../lib/round.js'
+
+const {
+  STORE_ALL = false
```

Comment:
```suggestion
  STORE_ALL = 'false'
```

Only strings can be passed as env vars, I think the boolean conversion should happen explicitly

Reply:
On the second thought, let's use `undefined` as the default value.